wallet/src/de/schildbach/wallet/ui/username/voting/RequestUsernameFragment.kt

```diff
@@ -49,9 +49,6 @@ class RequestUsernameFragment : Fragment(R.layout.fragment_request_username) {
 
         binding.usernameInput.doOnTextChanged { text, _, _, _ ->
             val username = text.toString()
-            binding.requestUsernameButton.isEnabled = username.isNotEmpty() &&
-                requestUserNameViewModel.canAffordNonContestedUsername()
-
```

Comment:
This was removed because it is handled in the uiState observer.  This code would allow the user to click "Request Username" before the validation of the name was completed.